Bubble_Sort/Bubble_Sort.kt

```diff
@@ -0,0 +1,30 @@
+//Function for Bubble Sort
+fun bubbleSort(list: MutableList<Int>):List<Int> {
+    // flag keeps track if any swap happened or not, if swap not happened then array is sorted and it will break out of the loop
```

Comment:
Please add more words of this line to the second line of comment.

Reply:
Done